### ISSUE TITLE
BUG: drizzle-kit introspect:mysql tables with pk and unique keys

### ISSUE THREAD

**Issue by bcerquiare** [drizzle-team/drizzle-kit-mirror#146]
Hello,
After updating Drizzle Kit to version 0.19.12, I've encountered errors with the `npx drizzle-kit introspect:mysql` command when working with tables that have unique keys.

When running the introspect command in Drizzle Kit, it doesn't place commas after the primaryKey column, which leads to an error because unique keys come right after the primary key:

```
export const table_name = mysqlTable("table_name", {
	column1: varchar("column1", { length: 65 }).notNull(),
	column2: varchar("column2", { length: 65 }).notNull(),
},
(table) => {
	return {
		table_name_column1 : primaryKey(table.column1)
		unq1: unique("unq1").on(table.unq_column2),
	}
});

```

I've configured my drizzle.config.ts to use the MySQL driver (mysql2)
![image](https://github.com/drizzle-team/drizzle-kit-mirror/assets/25101720/d7e21c0d-3465-46a0-bc8f-4dc96652f70a)

**Comment by PiotrekPKP** [comment]
Same issue!

**Comment by AndriiSherman** [comment]
should be fixed in `drizzle-kit@0.19.13`

**Comment by ViggoV** [comment]
I am having a very similar issue with drizzle-kit@0.20.14 and `introspect:pg` where a single index key also is missing a comma. As @wanjas suggests it may have something to do with sorting as the key would be last alphabetically, but the list seems to be sorted by length

![Screenshot 2024-04-17 at 09 59 40](https://github.com/drizzle-team/drizzle-kit-mirror/assets/3153568/5095af17-5c4e-4b74-a34e-93699c60a126)
